craco.config.js

```diff
@@ -15,5 +17,23 @@ module.exports = {
     alias: {
       '@mui/styled-engine': '@mui/styled-engine-sc',
     },
+    plugins: [
+      new CopyWebpackPlugin({
```

Comment:
It's been awhile since I've looked at webpack plugins, but I think instead of doing a blind `copy` of the config file into the build dir it might be possible to populate config values server-side and generate a "config" module that has the right values pre-populated from something like an `*.env` file. The benefits of doing it this way would be to save a round trip and make the initial time to render faster but if there are cons to this approach happy to discuss.

Reply:
We want to be able to configure the access time dropdowns. The first PR (https://github.com/discord/access/pull/194) used environment variables in `.env.production`. My attempt uses a config file because in the discord discussion (https://discord.com/channels/1232815453907189922/1306560215130443827) the proposed revision was to use a config file that would be copied in with webpack.

> populate config values server-side

Unfortunately, I don't understand what you want :(
I am not well-versed in React so that might be the problem.

Reply:
@savathoon please see my latest commit.

1. removed config files and config loader
2. create access_config.py to create a typed `AccessConfig` instance, looking for env vars that override individual properties.
3. created Flask endpoint in `app.py` to serve `/access_config.js`  
4. use `public/index.html` to load the `/access_config.js` script in every page
5. updated the front-end config loader to use `window.__ACCESS_CONFIG__` as the source of the config
6. updated the callsites

Please see what you think, because I'm not sure that I understand what you want.